src/styles/content/tables/styles.less

```diff
@@ -68,6 +68,7 @@
     // property. http://stackoverflow.com/questions/9789723/css-text-overflow-in-a-table-cell
     min-width: 0;
     overflow: hidden;
+    text-overflow: ellipsis;
```

Comment:
This will probably have implications elsewhere, but it's the right decision in my opinion.